guides/dsl.md

```diff
@@ -0,0 +1,241 @@
+# Eve JavaScript/TypeScript DSL
+
+The forthcoming v0.3 release of Eve supports a domain specific langauge (DSL) that interacts with the Eve server, allowing you to intermix Javascript and Eve code. There are several benefits to using the DSL:
```

Comment:
Yeah let's change "Eve server" to "Eve runtime" here to not give people the wrong idea. We do have a server, but it's a dumb server. It just manages getting files from crazy places into your browser.